.github/ISSUE_TEMPLATE/testing-checklist.md

```diff
@@ -12,8 +12,27 @@ assignees: ''
 
 *Use/modify this checklist as a more comprehensive list of core services to validate before cutting a major release.*
 
+## Test core services
+
+If the integration tests for all of the cloud providers are successful, that is a good sign!
+However, the following core services still need to be manually validated (until we can automate them).
+
+At minimum, the following services will need to be tested:
+- [ ] Log into keycloak as root user and add a user
+- [ ] Log into conda-store and create
+  - [ ] a conda environment in a shared namespace and,
+  - [ ] a conda environment in your personal namespace
+- [ ] Launch dask-gateway cluster, test auto-scaler and
+  - [ ] validate that the dask-labextention is working
+- [ ] Create a basic CDS Dashboard
+- [ ] Open VS-Code extension and create a python file
```

Comment:
```suggestion
- [ ] Open VS-Code 
  - [ ] add python extension
  - [ ] create a python file and run it
```

Reply:
Same comments throughout - need links (where do I get the Python extension from?)
If I have never used VSCode how do I run a file?